mcsema/BC/Function.cpp

```diff
@@ -618,7 +783,9 @@ static llvm::Function *LiftFunction(
   lifted_func->removeFnAttr(llvm::Attribute::AlwaysInline);
   lifted_func->removeFnAttr(llvm::Attribute::InlineHint);
   lifted_func->removeFnAttr(llvm::Attribute::NoReturn);
+  lifted_func->removeFnAttr(llvm::Attribute::NoUnwind);
   lifted_func->addFnAttr(llvm::Attribute::NoInline);
+  lifted_func->addFnAttr(llvm::Attribute::OptimizeNone);
```

Comment:
Why can't this be optimized?

Reply:
Will revert this change. Please refer the previous comment.